editor/block-mover/index.js

```diff
@@ -15,7 +15,8 @@ import { IconButton } from 'components';
 import './style.scss';
 import { isFirstBlock, isLastBlock } from '../selectors';
 
-function BlockMover( { onMoveUp, onMoveDown, isFirst, isLast } ) {
+// Only exported for testing.
```

Comment:
I'm a bit conflicted on including the comment. I think I might be motivated to avoid acknowledging the existence of tests from runtime code. The value of this named export isn't exclusive to tests only, but any consumer who'd want the presentational component. Then again, the distinction between importing a named export and the module default doesn't really clearly communicate this difference. It's rare though that we'd ever want to import the unconnected component in the UI, so this worry is not likely to come to light.

Which is all to say... I don't think we ought to include the comment, even though in all reality it truly is only exported for testing.